src/features/deepLinking/mocks/browser-window.ts

```diff
@@ -0,0 +1,128 @@
+import { EventEmitter } from 'events'
+
+import { WebContentsMock } from './web-contents'
+
+export class BrowserWindowMock extends EventEmitter {
```

Comment:
Btw, this should go to `<root>/__mocks__/electron.ts`, so it would be picked up by `jest` automatically. See how it's done here: https://github.com/pastelnetwork/pastel-electron-wallet/blob/393f8032c1c35678fa1ae502c962bde24c0bd551/__mocks__/electron.ts

Reply:
Yes, i have updated